strawberryfields/gbs/subgraph.py

```diff
@@ -19,25 +19,43 @@
 
 .. currentmodule:: strawberryfields.gbs.subgraph
 
-This module provides tools for users to identify dense subgraphs.
+This module provides tools for users to search for dense subgraphs of a range of different sizes.
```

Comment:
```suggestion
This module provides tools for users to search dense subgraphs of a range of different sizes.
```

Reply:
Is getting rid of the "for" correct here? "search dense subgraphs" (i.e., without the for) sounds like looking within the dense subgraphs themselves.

Reply:
How about changing "search" to "find"?

Reply:
Great!